Scripts/script-VMRayGetResults.yml

```diff
@@ -0,0 +1,46 @@
+commonfields:
+  id: 9364c36f-b1d6-4233-88c2-75008b106c31
+  version: 26
+name: vmray_getResults
+script: |-
+  import json
+  dArgs = {}
+  dArgs["sample_id"] = demisto.args()["sample_id"]
+  #res_jobs = json.loads(demisto.executeCommand("get_job_sample", dArgs)[0]['Contents'])
+  res_jobs = demisto.executeCommand("get_job_sample", dArgs)
+  # demisto.results(res_jobs)
+  if not res_jobs:
+      demisto.results("Hurray no jobs found, maybe the analysis is completed. Let's see!")
+      my_data = demisto.executeCommand("get_results", dArgs)
+      # my_data = json.loads(demisto.executeCommand("get_results", dArgs)[0]['Contents'])
+      # demisto.results(my_data)
+      parent_id = demisto.parentEntry()["id"]
+      # Das ist ein Kommentar 
```

Comment:
I don't know German but enough to understand that indeed, this is a comment (please remove this line and unused comments in line 14-15, 9, 11).